components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -423,6 +438,14 @@ - (void)setShowsHeaderDivider:(BOOL)showsHeaderDivider {
 
 #pragma mark - Dynamic Type
 
+- (void)setAdjustsFontForContentSizeCategory:(BOOL)adjustsFontForContentSizeCategory {
+  if (@available(iOS 10.0, *)) {
+    _adjustsFontForContentSizeCategory = adjustsFontForContentSizeCategory;
+    self.header.adjustsFontForContentSizeCategory = _adjustsFontForContentSizeCategory;
+    [self updateTable];
```

Comment:
Tip: Does the change in fonts require a full reload of data, or is it sufficient to just call `setNeedsLayout` on  `tableView`?